app/src/main/java/com/willowtree/vocable/settings/EditCategoriesViewModel.kt

```diff
@@ -56,6 +60,7 @@ class EditCategoriesViewModel : BaseViewModel() {
     fun deleteCategory(category: Category) {
         backgroundScope.launch {
             presetsRepository.deleteCategory(category)
+            // TODO: Delete category's phrases too
```

Comment:
We should make an issue for this if there isn't already

Reply:
Hmm so deleting the category is part of #239, should we split it out?

Reply:
Opened #276